lib/components/viewers/nearby/stop.tsx

```diff
@@ -19,12 +20,21 @@ const { getUserTimezone } = coreUtils.time
 export const fullTimestamp = (stoptime: StopTime) =>
   (stoptime.serviceDay || 0) + (stoptime.realtimeDeparture || 0)
 
+// Style for child stop headers
+const ChildStopHeader = styled.div`
+  background-color: #f5f5f5;
+  border-radius: 4px 4px 0 0;
+  font-size: 14px;
+  font-weight: bold;
+  padding: 10px 15px;
```

Comment:
Can we have the padding match the units we're using elsewhere? I think we're on units of 8